src/AElf.OS.Network.Grpc/Connection/IConnectionService.cs

```diff
@@ -7,7 +7,7 @@ namespace AElf.OS.Network.Grpc
     public interface IConnectionService
     {
         GrpcPeer GetPeerByPubkey(string pubkey);
-        Task DisconnectAsync(IPeer peer, bool sendDisconnect = false);
+        Task DisconnectAsync(IPeer peer, bool sendDisconnect = false, bool recover = true);
```

Comment:
DisconnectAsync with recover?

Reply:
Yes, not very clean, fixed, the new code does not use the event to schedule reconnections.